### PR TITLE
feat: Add last comment validator

### PR DESCRIPTION
**Pull request by cuong-tech**
Originally I thought this could be implemented as a filter (https://github.com/mergeability/mergeable/issues/639), but after giving it more thoughts, validator makes more sense.

The use case here is that developers can add a comment `merge` to trigger the `merge` action.

<details>
<summary>GIF</summary>

merge
</details>

Thanks to Bloomberg L.P. for letting us contribute to Mergeable.

### CODE REVIEW

File: __tests__/unit/actions/merge.test.js [hunk 1]
```diff
@@ -43,10 +19,15 @@ test('check that merge is called for status events', async () => {
   expect(context.octokit.pulls.merge.mock.calls[0][0].merge_method).toBe('merge')
 })
 
-test('check that merge is called for check suite events', async () => {
+test.each([
+  undefined,
+  'pull_request_review',
+  'check_suite',
+  'issue_comment'
+])('check that merge is called for %s events', async (eventName) => {
```
Comment (cuong-tech) [inline __tests__/unit/actions/merge.test.js:27]: small refactor to reduce code duplication

File: lib/eventAware.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ class EventAware {
       return context.payload
     }
 
-    if (context.eventName === 'issues') { // event name is 'issues' but payload contain 'issue'
+    if (context.eventName === 'issues' || context.eventName === 'issue_comment') {
```
Comment (cuong-tech) [inline lib/eventAware.js:22]: Remove comment as it seems a bit redundant

**Comment by cuong-tech** [comment]
@jusx @shine2lay could you take a look? 🙏

**Review by shine2lay — APPROVED** [review]
LGTM

**Pull request merged**